hack/build-cross.sh

```diff
@@ -22,7 +22,7 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-echo "NOTE: $0 has been replaced by 'make cross'"
+echo "NOTE: $0 has been replaced by 'make cross'. Run 'make help' for help."
```

Comment:
I don't think we need to modify these lines, as the specific command is already being called out.